src/index.js

```diff
@@ -32,8 +102,20 @@ function printComment(commentPath) {
   const comment = commentPath.getValue();
 
   switch (comment.ast_type) {
-    case "comment":
-      return comment.value;
+    case "comment": {
+      const hasNewLine = comment.value.lastIndexOf("\n");
+      let value = comment.value;
+      const hasSpaceAfterHash = value.charAt(1) === " ";
+      if (hasNewLine) {
+        value = value.slice(0, hasNewLine);
+      }
+
+      if (!hasSpaceAfterHash) {
+        value = "# " + value.slice(1, value.length);
+      }
+
+      return value;
```

Comment:
How about removing the leading `#` of the comment text, then stripping leading & trailing whitespace so there’s consistent spacing?

Reply:
Good idea 👍